example-clients/latent_client.c

```diff
@@ -21,30 +21,36 @@ jack_default_audio_sample_t *delay_line;
 jack_nframes_t delay_index;
 jack_nframes_t latency = 1024;
 
+#ifdef WIN32
+#define jack_sleep(val) Sleep((val))
```

Comment:
this macro is only used once, so I dont really think it makes sense. plus the -1*1000 for !win32 case makes it be very weird..

will remove this part to be more in line with the other examples, which do not use a macro

Reply:
Should we add a ticket for this?